jira/client.py

```diff
@@ -1635,14 +1640,15 @@ def comments(self, issue: str) -> List[Comment]:
         return comments
 
     @translate_resource_args
-    def comment(self, issue: str, comment: str) -> Comment:
+    def comment(self, issue: str, comment, expand=None):
```

Comment:
Type is removed here, it should stay in.

Reply:
Recommended update has been pushed.